sklearn/_min_dependencies.py

```diff
@@ -25,6 +25,7 @@
     "threadpoolctl": (THREADPOOLCTL_MIN_VERSION, "install"),
     "cython": (CYTHON_MIN_VERSION, "build"),
     "meson-python": ("0.16.0", "build"),
+    "ninja": ("1.11.0", "build"),
```

Comment:
Is the minimum version of Ninja documented somewhere? I guess the minimum version of Ninja might depend on the minimum version of Meson...